### ISSUE TITLE
Spectrum for D1245 missing

### ISSUE THREAD

**Issue by kelle** [elenamanjavacas/spectral_libraries#1]
the file for DENIS-P J124514.1-442907 is missing from the ISAAC tarball.

**Comment by elenamanjavacas** [comment]
True! I will update the .zip file and include this spectrum when I have a chance. Thanks for noticing!
